four-node-cluster/4-node-cluster.yml

```diff
@@ -832,6 +801,28 @@ Mappings:
       AMI: ami-0e81aa4c57820bb57
     us-west-2:
       AMI: ami-03fa1f014b48fa6bd
-Conditions:
-  IsMySQL: !Equals [!Ref DBType, mysql]
-  IsSQLServer: !Equals [!Ref DBType, mssql]
+  DBTypeMap:
+    mysql:
+      Version: '8.0.36'
+      License: 'general-public-license'
+      ParameterGroup: 'perf-test-param-group'
+      FromPort: '3306'
+      ToPort: '3306'
+      DBType: 'mysql'
+      DBName: 'WSO2_IS_DB_2'
+    mssql:
+      Version: '14.00.3480.1.v1'
+      License: 'license-included'
+      ParameterGroup: 'perf-test-param-group-sqlserver'
+      FromPort: '1433'
+      ToPort: '1433'
+      DBType: 'sqlserver-se'
+      DBName: ''
```

Comment:
why this doesn't have a db name?

Reply:
MSSQL databases does not support DB Name parameter